packages/kbn-dev-utils/src/code_ownership/shell_scripts/generate_code_owners.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+echo "### Generate Code Owners"
+echo ""
+
+CODEOWNERS_PATH=$1
+
+node scripts/generate_code_owners.js --verbose --codeOwnersPath $CODEOWNERS_PATH
```

Comment:
Can we just call this script directly? We don't need this wrapper right?

Reply:
Yeah I'm not sure where we wanna invoke this process.  I guess we could drop all invocations for now?
Perhaps it should be invoked in kibanaPipeline.groovy?